### ISSUE TITLE
[iceprod2] redo db tests (Trac #1433)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1425]
Once TestDB is finished (#1424), redo all the db tests:

dbmethods_auth_test.py - juancarlos
dbmethods_cron_test.py - david.schultz
dbmethods_misc_test.py - obsolete
dbmethods_node_test.py - juancarlos
dbmethods_queue_test.py - david.schultz
dbmethods_rpc_test.py - ddelventhal
dbmethods_web_test.py - ddelventhal

Migrated from https://code.icecube.wisc.edu/ticket/1433
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:28", 
    "description": "Once TestDB is finished (#1432), redo all the db tests:\n\ndbmethods_auth_test.py - juancarlos\ndbmethods_cron_test.py - david.schultz\ndbmethods_misc_test.py - obsolete\ndbmethods_node_test.py - juancarlos\ndbmethods_queue_test.py - david.schultz\ndbmethods_rpc_test.py - ddelventhal\ndbmethods_web_test.py - ddelventhal", 
    "reporter": "david.schultz", 
    "cc": "juancarlos, ddelventhal", 
    "resolution": "fixed", 
    "_ts": "1550067088921308", 
    "component": "iceprod", 
    "summary": "[iceprod2] redo db tests", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-11-11T20:56:46", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20151112T18:49:32: 
* david.schultz commented:

> Commence the redoing!


* david.schultz changed description from:

> Once TestDB is finished (#1432), redo all the db tests:
> 
> dbmethods_auth_test.py - juancarlos
> dbmethods_cron_test.py - david.schultz
> dbmethods_misc_test.py - juancarlos
> dbmethods_node_test.py - juancarlos
> dbmethods_queue_test.py - david.schultz
> dbmethods_rpc_test.py - ddelventhal
> dbmethods_web_test.py - ddelventhal

to:

> Once TestDB is finished (#1432), redo all the db tests:
> 
> dbmethods_auth_test.py - juancarlos
> dbmethods_cron_test.py - david.schultz
> dbmethods_misc_test.py - obsolete
> dbmethods_node_test.py - juancarlos
> dbmethods_queue_test.py - david.schultz
> dbmethods_rpc_test.py - ddelventhal
> dbmethods_web_test.py - ddelventhal

**Comment by icecube-trac** [comment]
Trac update at 20151112T20:10:35: david.schultz commented:

> as a note to help run only one test:
> 
> ```text
> ./coverage.sh --server dbmethods_queue *get_site*
> ```

**Comment by icecube-trac** [comment]
Trac update at 20151117T16:59:30: david.schultz commented:

> completed:
> 
> dbmethods_cron_test.py
> dbmethods_queue_test.py

**Comment by icecube-trac** [comment]
Trac update at 20151204T21:52:43: 
* david.schultz commented:

> In changeset 140112:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="140112"
> fix the last of the dbmethods tests. closes #1425
> ```


* david.schultz changed resolution from "" to "fixed"
* david.schultz changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:28: 
* olivas commented:

> Milestone IceProd 2.0 deleted


* olivas changed milestone from "IceProd 2.0" to ""
